utils/format-assets.js

```diff
@@ -7,41 +7,34 @@ function getAssets(stats) {
 }
 
 function getAssetSize(asset) {
-  return `${filesize(asset.size)}${asset.minGz && " (min+gz)" || ""}`;
+  return `${filesize(asset.size)}${(asset.minGz && " (min+gz)") || ""}`;
 }
 
 function getTotalSize(assets) {
-  return filesize(assets.reduce(
-    (total, asset) => total + asset.size,
-    0
-  ));
+  return filesize(assets.reduce((total, asset) => total + asset.size, 0));
 }
 
 function resolveAssets(tree, bundles) {
   return _.flatMap(assets =>
-    assets
-      .filter(asset => asset.name.indexOf("hot-update") < 0)
-      .map(asset => {
-        const realBundleMatch = _.find({ path: asset.name })(bundles);
-        return realBundleMatch ? {
-          name: realBundleMatch.path,
-          size: realBundleMatch.metrics.meta.bundle.minGz,
-          minGz: true
-        } : asset;
-      })
+    assets.filter(asset => asset.name.indexOf("hot-update") < 0).map(asset => {
```

Comment:
Hmmm... prettier newbie question -- any way to be able to do something like:

```js
assets
  .filter(
  .map(
````

etc? I like having each part of an array chain be newlined...

Reply:
It’s prettier when not using prettier 😂